pkg/kubectl/stop.go

```diff
@@ -88,18 +91,25 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string, timeout
 	if err != nil {
 		return "", err
 	}
+	ctrl, err := rc.Get(name)
+	if err != nil {
+		return "", err
+	}
 	if timeout == 0 {
-		rc, err := rc.Get(name)
-		if err != nil {
-			return "", err
-		}
-		timeout = Timeout + time.Duration(10*rc.Spec.Replicas)*time.Second
+		timeout = Timeout + time.Duration(10*ctrl.Spec.Replicas)*time.Second
 	}
-	retry := NewRetryParams(reaper.pollInterval, reaper.timeout)
-	waitForReplicas := NewRetryParams(reaper.pollInterval, timeout)
-	if err = scaler.Scale(namespace, name, 0, nil, retry, waitForReplicas); err != nil {
+	overlappingCtrl, err := getReplicationControllersForLabels(rc, labels.Set(ctrl.Spec.Selector))
+	if err != nil {
 		return "", err
 	}
+	sort.Sort(controller.OverlappingControllers(overlappingCtrl))
+	if overlappingCtrl[0].Name == name {
```

Comment:
Quoting @bgrant0607: "Resizing the active controller down to zero and then allowing the next one in the sequence to scale up again would probably be surprising to users."

That's what this will do, isn't it?